src/stores/userStore.ts

```diff
@@ -246,6 +246,12 @@ export const useUserStore = defineStore('user', () => {
     return userCookieValue.email
   }
 
+  function getCookieProfileProvider(): string {
+    // Cannot be a computed value as cookies are not reactive
+    const userCookieValue = getUserCookieValue()
+    return userCookieValue.profile.provider
```

Comment:
```suggestion
    return userCookieValue.profile?.provider
```

Let's protect ourselves from reading an attribute of undefined

Reply:
Done